util/flags.py

```diff
@@ -135,9 +135,15 @@ def create_flags():
     # Early Stopping
 
     f.DEFINE_boolean('early_stop', True, 'enable early stopping mechanism over validation dataset. If validation is not being run, early stopping is disabled.')
-    f.DEFINE_integer('es_steps', 4, 'number of validations to consider for early stopping. Loss is not stored in the checkpoint so when checkpoint is revived it starts the loss calculation from start at that point')
-    f.DEFINE_float('es_mean_th', 0.5, 'mean threshold for loss to determine the condition if early stopping is required')
-    f.DEFINE_float('es_std_th', 0.5, 'standard deviation threshold for loss to determine the condition if early stopping is required')
+    f.DEFINE_integer('es_epochs', 4, 'Number of epochs with no improvement after which training will be stopped. Loss is not stored in the checkpoint so when checkpoint is revived it starts the loss calculation from start at that point')
```

Comment:
How about just tying this number to `--plateau_epochs`? Just make it always be 2x that flag. Avoids configuration errors and it's a pretty good default I would say.

Reply:
I also thought about it. The downside is that you cannot customise it anymore. But i also dont know where this really makes a difference.

By the way, in keras early stopping and rlrop are two different callbacks.

Reply:
I think a factor of 2 may be too small, because then the learning rate will be reduced only once before doing the early stop.